ui.tests/test-module/specs/formcontainer.spec.js

```diff
@@ -287,4 +287,27 @@ describe('Page/Form Authoring', function () {
             });
 
         });
+
+        context("Check default behaviour in Form Editor", function () {
+            const pagePath = "/content/forms/af/core-components-it/samples/emailinput/basic.html";
+
+            beforeEach(function () {
+                cy.previewForm(pagePath);
+            });
+
+            it('check the preventDefaultSubmit method by simulating keydown event on the form', function () {
+                cy.get('.cmp-adaptiveform-container').then((formContainer) => {
+                    cy.stub(formContainer[0], 'onsubmit').as('submit');
+                    cy.get('form').trigger('keydown', {key: 'Enter'});
```

Comment:
trigger"enter" on button and input as well for completeness of testing

Reply:
Added these in the check below.